Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
-name = "imap-proto"
-version = "0.1.0"
 authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
+categories = ["authentication", "email", "encoding", "network-programming", "parser-implementations"]
```

Comment:
I don't think this crate fits in with the other crates in `authentication` and `encoding`. `network-programming` and `parser-implementations` look like fine additions, though!

Reply:
Removed the extra categories.